README.md

```diff
@@ -84,14 +84,32 @@ When the configuration loader starts, it looks in these locations in this order,
 
 ## Using Mycroft Without Home
 
-If you do not wish to use the Mycroft Home service, you may insert your own API keys into the configuration files listed below in <b>configuration</b>.
+If you do not wish to use the Mycroft Home service, before starting Mycroft for the first time, create `$HOME/.mycroft/mycroft.conf` with the following contents:
 
-The place to insert the API key looks like the following:
+```
+{
+  "skills": {
+    "blacklisted_skills": [
+      "mycroft-configuration.mycroftai",
+      "mycroft-pairing.mycroftai"
+    ]
+  }
+}
+```
+
+Mycroft will then be unable to perform speech-to-text conversion, so you'll need to set that up as well, using one of the STT engines Mycroft supports.
```

Comment:
Maybe we should link to the STT setup section of the documentation here: https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/stt-engine

Reply:
Good idea; updated